app/views/challenges/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Challenges</h1>
+<ul>
+  <% @challenges.each do |challenge| %>
+    <p>Challenge Theme: <a href="<%= challenge_path(challenge) %>"><%= challenge.theme %></a>
```

Comment:
Nice job using `challenge_path`! I would try to use `link_to` instead of the HTML `<a>` tag more in the future.